src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -2965,6 +3139,60 @@ private void processFileChanges(
         // reset this property in case it had been set to true
         System.setProperty(SKIP_BETA_INSTALL_WARNING, Boolean.FALSE.toString());
 
+        // upstream project source file changed
+        if (!this.upstreamProjects.isEmpty()) {
```

Comment:
Should this check for null? e.g. `if (upstreamProjects != null && !upstreamProjects.isEmpty()) {`